configuration.sh

```diff
@@ -3,4 +3,5 @@
 brightFolder="/sys/class/backlight/intel_backlight/"
 currentBright=`cat $brightFolder"brightness"`
 maxBright=`cat $brightFolder"max_brightness"`
+minBright=$((maxBright / 20))
```

Comment:
Hi, thanks for the PR, quick question: Why is minBright a twentieth of maxbright?
I'd assume this is 0 or catted from brightFolder.

Reply:
Right I see you want to keep it above 0 so you can still see the screen.

Reply:
Yeah, it should always be above 0, and I thought 1/20 (5%) would be a good value.